<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mobilenumbers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>chicken-emblem_10272-25.jpg</normaloff>chicken-emblem_10272-25.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.046, y1:0.08, x2:1, y2:1, stop:0 rgba(241, 241, 241, 255), stop:0.55 rgba(203, 203, 203, 255), stop:0.954545 rgba(155, 155, 155, 255), stop:0.971591 rgba(255, 255, 0, 255), stop:0.977273 rgba(189, 189, 189, 255), stop:1 rgba(189, 189, 189, 255));</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>400</width>
      <height>371</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>197</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>32</number>
    </attribute>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>الاسم</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>الرقم</string>
     </property>
    </column>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>133</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>الاسم</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>الرقم</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>اضافة</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>add.png</normaloff>add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(37, 167, 224);</string>
      </property>
      <property name="text">
       <string>بحث</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>seawrch.png</normaloff>seawrch.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>27</width>
        <height>29</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(14, 209, 69);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>حذف</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>delete.png</normaloff>delete.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>27</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>تحديث</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>uptate2.png</normaloff>uptate2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
